public/app/common/services/devices/discovery-service/discovery-service.js

```diff
@@ -1,4 +1,4 @@
-var Rx = require('rx'),
+var Rx = require('rx/dist/rx.all.min'),
```

Comment:
We do not have `uglify` plugin in webpack yet.
This change was a test to see what changes will be made to the bundle.
Just ignore it for now (it should be done in the webpack config globally, and not in the code)